docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -174,7 +213,7 @@ If the properties panel for your form doesn't open automatically, navigate to **
 
 ## Step 3: Link the form to your process
```

Comment:
If you create a form according to these instructions, then this step is automatically completed.